Cargo.toml

```diff
@@ -882,7 +880,7 @@ minimal-template-node = { path = "templates/minimal/node" }
 minimal-template-runtime = { path = "templates/minimal/runtime" }
 mixnet = { version = "0.7.0" }
 mmr-gadget = { path = "substrate/client/merkle-mountain-range", default-features = false }
-mmr-lib = { version = "0.5.2", package = "ckb-merkle-mountain-range" }
+mmr-lib = { version = "0.8.1", package = "polkadot-ckb-merkle-mountain-range", default-features = false }
```

Comment:
Before this change, `pallet-bridge-beefy` used the original `ckb-merkle-mountain-range`, and `sp-mmr` used the' polkadot` fork.
I unified them

Reply:
cc @serban300

Reply:
Good catch ! Thanks ! Yes, we should use the `polkadot` fork everywhere.